services/storage-users/pkg/command/trash_bin.go

```diff
@@ -1,23 +1,60 @@
 package command
 
 import (
+	"context"
+	"fmt"
+	"path"
+	"path/filepath"
+	"slices"
+	"strings"
 	"time"
 
+	gateway "github.com/cs3org/go-cs3apis/cs3/gateway/v1beta1"
+	rpc "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
+	provider "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
 	"github.com/cs3org/reva/v2/pkg/events"
+	"github.com/cs3org/reva/v2/pkg/rgrpc/todo/pool"
+	"github.com/cs3org/reva/v2/pkg/storagespace"
+	"github.com/cs3org/reva/v2/pkg/utils"
+	"github.com/mohae/deepcopy"
 	"github.com/owncloud/ocis/v2/ocis-pkg/config/configlog"
+	"github.com/owncloud/ocis/v2/ocis-pkg/tracing"
 	"github.com/owncloud/ocis/v2/services/storage-users/pkg/config"
 	"github.com/owncloud/ocis/v2/services/storage-users/pkg/config/parser"
 	"github.com/owncloud/ocis/v2/services/storage-users/pkg/event"
+	"github.com/owncloud/ocis/v2/services/storage-users/pkg/logging"
+	"github.com/rs/zerolog/log"
 	"github.com/urfave/cli/v2"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 )
 
+const (
+	SKIP = iota
+	REPLACE
+	KEEP_BOTH
+
+	retrievingErrorMsg = "trash-bin items retrieving error"
+)
+
+var optionFlagTmpl = cli.StringFlag{
+	Name:        "option",
+	Value:       "skip",
+	Aliases:     []string{"o"},
+	Usage:       "The restore option defines the behavior if the file with the same name that needs to be restored already exists in a space. Expected values [skip] [replace] [keep-both]",
+	DefaultText: "The default value is [skip] restoring exists file. ",
```

Comment:
```suggestion
	Usage:       "The restore option defines the behavior for a file to be restored, where the file name already already exists in the target space. Expected values [skip] [replace] [keep-both]",
	DefaultText: "The default value is [skip] restoring an existing file. ",
```
If committed, needs text adaption in the readme above too.